components/xperiencify/sources/student-added-to-course/student-added-to-course.ts

```diff
@@ -0,0 +1,57 @@
+import { defineSource } from "@pipedream/types";
+import xperiencify from "../../app/xperiencify.app";
+
+export default defineSource({
+  name: "Student Added to Course",
+  description: "Emit new event when a student enrolls into a course.",
+  key: "xperiencify-student-added-to-course",
+  version: "0.0.1",
+  type: "source",
```

Comment:
Shouldn't you add `dedupe` strategy here?
```suggestion
  type: "source",
  dedupe: "unique",
```